content/docs/user-guide/components/reference/paintbrush/paintbrush.md

```diff
@@ -0,0 +1,169 @@
+| Hardness 50% | ![50% Hardness 1% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-50-flow-1.png) | ![50% Hardness 50% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-50-flow-50.png) | ![50% Hardness 100% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-50-flow-100.png) |
+| Hardness 100% | ![100% Hardness 1% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-100-flow-1.png) | ![100% Hardness 50% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-100-flow-50.png) | ![100% Hardness 100% Flow.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-100-flow-100.png) |
+
+Because the Hardness opacity affects each brush circle, overlapping circles in the same brush stroke will blend with each other.
+
+An example of a brush stroke crossing itself with medium hardness:![Example of brush stroke crossing itself with medium hardness.](/images/user-guide/components/reference/paintbrush/paintbrush-hardness-overlap.png)
+
+#### Distance
+
+**Distance** controls the spacing of each circle within the brush stroke as the mouse moves. Very specifically, The Distance value represents the total distance the mouse needs to move before applying another circle, so small back-and-forth movements with the mouse can produce circles that are closer together than the Distance value. The Distance is a percentage of the circle's size, so a value of 50% will overlap each circle by 50% and a value of 100% will produce exactly non-overlapping circles. Values greater than 100% will leave space between each circle.
+
+| Distance | Illustration |
+| - | - |
+| 1% | ![Paintbrush distance of 1%.](/images/user-guide/components/reference/paintbrush/paintbrush-distance-1.png) |
+| 25% | ![Paintbrush distance of 25%.](/images/user-guide/components/reference/paintbrush/paintbrush-distance-25.png) |
+| 50% | ![Paintbrush distance of 50%.](/images/user-guide/components/reference/paintbrush/paintbrush-distance-50.png) |
+| 100% | ![Paintbrush distance of 100%.](/images/user-guide/components/reference/paintbrush/paintbrush-distance-100.png) |
+| 200% | ![Paintbrush distance of 200%.](/images/user-guide/components/reference/paintbrush/paintbrush-distance-200.png) |
+
+### Settings that apply to smoothing operations
+
+The following settings only apply to the smoothing brush.
+
+#### Smooth Mode
+
+**Smooth Mode** is the method for combining an NxN set of values together into a single smoothed value for use at the center of the NxN square.
+
+| Mode | Description | Example 1 | Example 2 |
+| - | - | - | - |
+| (None) | (Input images used in smoothing) |![Sample smoothing input image.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-input.png) |![Sample smoothing input image 2.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-input2.png) |
```

Comment:
```suggestion
| `None` | Input images used in smoothing. |![Sample smoothing input image.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-input.png) |![Sample smoothing input image 2.](/images/user-guide/components/reference/paintbrush/paintbrush-smooth-mode-input2.png) |
```
If these are enumerated values of a property, they should use code style.  Should this description read "The input images use no smoothing."?

Reply:
So...maybe this table isn't doing what I had hoped. The "(None)" option _isn't_ an enumerated value. That row is here to provide the baseline of what things look like without smoothing, so that every image below can be compared to it to see how it changes with the different options. All of the other entries are the enumerated values.

Is there a better way to present this information that would be less confusing?

Reply:
OK, I didn't realize that.  At the very least, you can drop 'None' entirely and leave that cell of the table empty.  I'm fine with either your original or my suggestion for the Description cell.  Maybe just (Input images)?  Your call.

Reply:
Oooh, good idea. I'll drop the 'None' and change the Description to '(Input images)'.